templates/puppetboard-ingress.yaml

```diff
@@ -1,8 +1,10 @@
-{{- if .Values.puppetboard.enabled }}
-{{- if .Values.puppetboard.ingress.enabled }}
+{{- if and .Values.puppetboard.enabled .Values.puppetboard.ingress.enabled }}
 {{- $releaseName := .Release.Name -}}
 {{- $serviceName := "puppetdb" }}
 {{- $servicePort := .Values.puppetboard.port -}}
+{{- $pathType := .Values.puppetboard.ingress.pathType | default "ImplementationSpecific" -}}
+{{- $apiIsStable := eq (include "puppetserver.ingress.isStable" .) "true" -}}
```

Comment:
Not clear what this is used for.

Reply:
Fixed - for optional ingress class name. Thanks.